### PR TITLE
Term entry c sharp cos

### LINKED ISSUE #1615: [Term Entry] C# Math Functions `.Cos()`
### Reason/inspiration (optional)

We would like a new term entry for [C#](https://www.codecademy.com/resources/docs/c-sharp) about the `Math.Cos()` method.

The entry should go in a new file in `docs/content/c-sharp/concepts/math-functions/terms/cos/cos.md`

The entry should

- Have a syntax section illustrating the method with a ` ```pseudo ` code block.
- An example showing the method in use with a ` ```cs ` code block.
- A runnable codebyte example in a ` ```codebyte/csharp ` block.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by cslylla**
### Description

- Added new term for C# Math Functions: Math.Cos()
- Closes #1615 

### Type of Change

- Adding a new entry

### Checklist

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [ ] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/c-sharp/concepts/math-functions/terms/cos/cos.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+The **`Math.Cos()`** class method returns the cosine of a given angle.
+
+## Syntax
+
+```pseudo
+Math.Cos(angle);
+```
+
+The `Math.Cos()` method takes only one `double` type parameter - `angle` - an angle in radians. The `Math.Cos()` method returns the cosine of the `angle` as a `double` value or `NaN` (not a number) if the `angle` equals to:
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/cos/cos.md:25]:
```suggestion
The `Math.Cos()` method takes only one parameter,`angle`, an angle in radians of type `double`. The method returns the cosine of the `angle` as a `double` value or `NaN` (not a number) if the value of `angle` equals:
```

File: content/c-sharp/concepts/math-functions/terms/cos/cos.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+- `NaN`,
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/cos/cos.md:27]:
```suggestion
- `NaN`
```

File: content/c-sharp/concepts/math-functions/terms/cos/cos.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+- `NegativeInfinity`, or
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/cos/cos.md:28]:
```suggestion
- `NegativeInfinity`
```

File: content/c-sharp/concepts/math-functions/terms/cos/cos.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+- `PositiveInfinity`.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/cos/cos.md:29]:
```suggestion
- `PositiveInfinity`
```

File: content/c-sharp/concepts/math-functions/terms/cos/cos.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+
+    Console.WriteLine("The cosine of " + degrees + " degrees is: " + cosine);
+  }
+}
+```
+
+The example will result in the following output:
+
+```shell
+The cosine of 60 degrees is: 0.5
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/cos/cos.md:53]: I get something like: .50000001. Perhaps round to one or two decimal places before writing to the console

File: content/c-sharp/concepts/math-functions/terms/cos/cos.md [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+
+The following example is runnable and returns the cosine of the `angle` given in degrees:
+
+```codebyte/csharp
+using System;
+
+public class Example {
+
+  public static void Main(string[] args) {
+    //angle in degrees
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/cos/cos.md:66]:
```suggestion
    // Angle in degrees
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla Please update and tag me when you've finished making changes. Thanks

File: content/c-sharp/concepts/math-functions/terms/cos/cos.md (hunk 5 above)
Comment (cslylla) [reply content/c-sharp/concepts/math-functions/terms/cos/cos.md:53]: I added Math.Round(), but I still get 0.5. The cosine of 60 degrees should be 0.500000. 
cos
Do you know what the issue could be?

**Comment by cslylla** [comment]
@caupolicandiaz I accepted the changes and added rounding of the result to the example, but I still could not solve the issue.

File: content/c-sharp/concepts/math-functions/terms/cos/cos.md [hunk 7]
```diff
@@ -22,15 +22,17 @@ The **`Math.Cos()`** class method returns the cosine of a given angle.
...
+The `Math.Cos()` method takes only one parameter, `angle`, an angle in radians of type `double`. The method returns the cosine of the `angle` as a `double` value or `NaN` (not a number) if the value of `angle` equals:
 
-- `NaN`,
-- `NegativeInfinity`, or
-- `PositiveInfinity`.
+- `NaN`
+- `NegativeInfinity`
+- `PositiveInfinity`
+
+> **Note:** Depending on the used operating system or architecture, the exact result or the input range of the `Math.Cos()` method may differ due to the differences in the C runtime environment.
```
Comment (caupolicandiaz) [inline content/c-sharp/concepts/math-functions/terms/cos/cos.md:31]: I think we should move the note below the example text (after line 35). Also with the note we can forego the rounding step.

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@cslylla Just a couple of additional minor edits.

**Comment by cslylla** [comment]
@caupolicandiaz I moved the note and removed the rounding from the examples.

**Review by caupolicandiaz — APPROVED** [review]
@cslylla This is ready for another review. As noted in the sine entry I elected to forego the runtime note.

**Review by SSwiniarski — APPROVED** [review]
@cslylla Looks good to merge.

**Comment by SSwiniarski** [comment]
@cslylla Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/c-sharp/math-functions/cos

image

**Pull request merged**